### PR TITLE
[#140] Add version bumping workflow to generated project

### PR DESCRIPTION
**Pull request by chornerman**
- Solves #140 

## What happened 👀

- Refactor version bumping workflow for the template
- Create version bumping workflow for a generated project

## Insight 📝

N/A

## Proof Of Work 📹

Workflow still runs normally: https://github.com/nimblehq/flutter_templates/pull/171

### CODE REVIEW

File: template/.github/workflows/bump_version.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set new version
+        id: set_version
+        run: |
+          perl -i -pe 's/^(version:\s+\d+\.\d+\.\d+\+)(\d+)$/"version: ${{ github.event.inputs.newVersion }}+".($2+1)/e' ./template/pubspec.yaml
```
Comment (luongvo) [inline template/.github/workflows/bump_version.yml:24]: we need to update the path. Tt's the generated project's workflow, the path must not contain `/template`

```suggestion
          perl -i -pe 's/^(version:\s+\d+\.\d+\.\d+\+)(\d+)$/"version: ${{ github.event.inputs.newVersion }}+".($2+1)/e' ./pubspec.yaml
```

**Review by luongvo — CHANGES_REQUESTED** [review]
The rest lgtm

File: template/.github/workflows/bump_version.yml (hunk 1 above)
Comment (chornerman) [reply template/.github/workflows/bump_version.yml]: 😮 Thanks for spotting out

**Review by luongvo — APPROVED** [review]
LGTM

**Comment by luongvo** [comment]
@chornerman please rebase 👋

**Review by doannimble — APPROVED** [review]
LGTM 🚀

**Pull request merged**